smoke-tests/smoke-tests.gradle

```diff
@@ -6,18 +6,18 @@ dependencies {
   api deps.spock
   api project(':testing-common')
 
-  implementation project(':auto-exporters:opentelemetry-auto-exporter-logging')
+  implementation project(':javaagent-exporters:logging')
   implementation deps.slf4j
   implementation deps.opentelemetryProto
   implementation "org.testcontainers:testcontainers:1.14.3"
   implementation 'com.google.protobuf:protobuf-java-util:3.12.4'
 }
 
 test {
-  dependsOn = [':javaagent:shadowJar', ':auto-exporters:opentelemetry-auto-exporter-logging:shadowJar']
+  dependsOn = [':javaagent:shadowJar', ':javaagent-exporters:logging:shadowJar']
 
   doFirst {
     jvmArgs "-Dio.opentelemetry.smoketest.agent.shadowJar.path=${project(':javaagent').tasks.shadowJar.archivePath}"
-    jvmArgs "-Dotel.exporter.jar=${project(':auto-exporters:opentelemetry-auto-exporter-logging').tasks.shadowJar.archivePath}"
+    jvmArgs "-Dotel.exporter.jar=${project(':javaagent-exporters:logging').tasks.shadowJar.archivePath}"
```

Comment:
That is now unused, btw. Seems like I forgot to delete. All smoke tests should now use Collector.